yt/data_objects/construction_data_containers.py

```diff
@@ -2675,6 +2714,8 @@ def export_sketchfab(
             color_log,
             sample_type="vertex",
             no_ghost=no_ghost,
+            color_field_max,
+            color_field_min,
```

Comment:
@evanoconnor The error you're getting is indeed related to the use of `*,`

> positional argument follows keyword argument

In Python, some arguments may be passed positionally (for instance in `abs(-2)`), and others may be passed by name (as 'keywords') (for instance in `range(0, 10, step=2)`, `step` is passed as keyword). However what's not allowed is passing positional arguments *after* keyword ones.

Here `sample_type` and `no_ghost` are passed as keywords, so it's invalid to pass `color_field_max/min` positionally as is currently done. To fix it, pass these args as keywords:
```suggestion
            color_field_max=color_field_max,
            color_field_min=color_field_min,
```

Is this explanation clear ?

Reply:
oh course! sorry.

So, for `_export_ply`:

```
def _export_ply(
        self,
        filename,
        bounds=None,
        color_field=None,
        color_map=None,
        color_log=True,
        sample_type="face",
        *,
        color_field_max=None,
        color_field_min=None,
    ):
```
 there is a `*,` but since the call to it:

```
       self._export_ply(
            filename,
            bounds,
            color_field,
            color_map,
            color_log,
            sample_type,
            color_field_max,
            color_field_min
        )
```
 didn't specify any keyword arguments it was fine? even with the `*,`? Would you suggest adding 

```
            color_field_max=color_field_max,
            color_field_min=color_field_min
```

here too?

Reply:
Yes, that's actually necessary, since the whole point of using `, *` is to _forbid_ passing arguments positionally :)